### PR TITLE
Display NOK PrestaShop requirements

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Display at least the check "name" when a PrestaShop prerequisite is non-ok, event if there is no description text. This PR can be completed by covering all checks, its main purpose is to always display an information to the user, even incomplete.
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | Nope
| Deprecations? | Nope
| Fixed ticket? | http://build.prestashop.com/news/coreweekly-week-30-2018/#comment-4017968715
| How to test?  | Disable the fileinfo extension and reach the information page in your back office. Without the PR, the list is empty. With it, you know at least what was the test made.

capture du 2018-08-06 17-36-59

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9407)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]: Must be in _Admin.Advparameters.Feature_.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig
Comment (LouiseBonnard) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]: Must be in _Admin.Advparameters.Feature_.

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig
Comment (mickaelandrieu) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]: I'm not sure we should patch this in the template, but ensure the key is in requirements.testsError array, don't you think?

File: src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig
Comment (mickaelandrieu) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]: Wy? it's a notification :( (answering to @LouiseBonnard)

Comment (LouiseBonnard) [reply src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/system_information.html.twig]: Because we classify the basic content of a page in the Feature file.

**Comment by mickaelandrieu** [comment]
While I agree it's better to display an incomplete information than nothing, I'd like to see this hack outside of Twig template.

Can't this be done before in the application workflow?

**Review by mickaelandrieu — DISMISSED** [review]
Thanks !

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Quetzacoalt91** [comment]
One more review from @LouiseBonnard and we're good. I saw 2 domains were wrong.

**Comment by mickaelandrieu** [comment]
so be it, I can't merge without @LouiseBonnard approval ^^

File: src/Adapter/Requirement/CheckRequirements.php
Comment (LouiseBonnard) [inline src/Adapter/Requirement/CheckRequirements.php]: Sorry guys for the delayed review, long weekend... Can't we write _Set write permissions_ instead of _Set the write permissions_ to have all the wordings harmonized? Otherwise domains are good here. ;-)

Comment (Quetzacoalt91) [reply src/Adapter/Requirement/CheckRequirements.php]: Nice catch, updated.

**Review by mickaelandrieu — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @Quetzacoalt91 and everyone for reviews :)

**Comment by Prestapro** [comment]
Hello,

I have the same error on 2 different stores, do you have a solution?

**Comment by Quetzacoalt91** [comment]
You can temporarily apply the changes suggested in https://github.com/PrestaShop/PrestaShop/pull/9407/commits/bdd7bb49db96ba40d79003482b29fa8bcaf9316b.

The actual error will appear on the page.

**Comment by rdy4ever** [comment]
Works great! Thanks

**Pull request merged**
